source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/en-us.json

```diff
@@ -25,7 +25,9 @@
     "none": "None",
     "abbreviation": "Abbreviation",
     "general": "General Information",
-    "notAvailable": "Not Available"
+    "notAvailable": "Not Available",
+    "archive": "Archive",
```

Comment:
I thought we already had archive somewhere?

Reply:
Probably I wanted to change to common since it's an action similar to save or add, I will check where is the other archives to refactor them